Makefile

```diff
@@ -299,20 +298,23 @@ update-go-version:
 goversioncheck:
 	./verify-go-version.sh
 
-TRAY_RELEASE ?= packaging/tmp/crc-tray-macos-$(GOARCH).tar.gz
+TRAY_RELEASE ?= packaging/tmp/crc-tray-macos-universal.tar.gz
 
 embed-download: $(HOST_BUILD_DIR)/crc-embedder
 ifeq ($(CUSTOM_EMBED),false)
 	mkdir -p $(EMBED_DOWNLOAD_DIR)
 	$(HOST_BUILD_DIR)/crc-embedder download $(EMBED_DOWNLOAD_DIR)
 endif
 
-packaging/vfkit-$(GOARCH).entitlements:
+packaging/vfkit-universal.entitlements:
 	curl -sL https://raw.githubusercontent.com/code-ready/vfkit/main/vf.entitlements -o $@
 
-packagedir: clean embed-download macos-release-binary packaging/vfkit-$(GOARCH).entitlements
+prepare_crc_fat_binary: macos-release-binary
+	mkdir -p out/macos-universal
+	lipo -create out/macos-amd64/crc out/macos-arm64/crc -output out/macos-universal/crc
```

Comment:
`lipo` is a macos binary. I think there's a reimplementation of it in go, dunno if you tried it? I'm wondering whether we should be generating this universal binary as part of the signing process or during the cross build on linux?

Reply:
This can be adjusted later though